src/extensions/index.js

```diff
@@ -1,5 +1,5 @@
-export { MjmlComment } from './mjml-comment';
-export { MjmlConditionalComment } from './mjml-conditional-comment';
```

Comment:
prettier adds a lot of noise to this PR. Is it possible/feasibly to run prettier as a separate PR on these files?

Reply:
It should be, I'll try to remove the noisy prettier updates

Reply:
Issue was that the generate-mjml-react script ran prettier on the src folder after completion. I updated this to only run on src/index.tsx and src/mjml